src/browser-frontend/views/browser/chrome/tabbar/new-tab-button.jsx

```diff
@@ -43,9 +43,6 @@ export default class NewTabButton extends PureComponent {
           height="12px"
           styleName="icon"
         />
-        <div>
-          New Tab
-        </div>
```

Comment:
With this change, the new tab button contents aren't properly horizontally aligned in the button.
<img width="129" alt="screen shot 2017-05-12 at 2 28 53 pm" src="https://cloud.githubusercontent.com/assets/248899/25998513/a2be75ba-3721-11e7-97c9-3581af1bd8ac.png">